api/src/models/api/INetworkGetResponse.ts

```diff
@@ -75,4 +75,8 @@ interface INetworkView {
      * If Identity Resolver tool should be supported.
      */
     identityResolverEnabled?: boolean;
+    /**
+     * Max results for API requests.
+     */
+    apiMaxResults?: number;
```

Comment:
Should we call it `legacyApiMaxResults` or something ? 😅

Reply:
we can leave it as is, but if we do we have to add to the description that for now this is implemented in legacy networks only